packages/connect-examples/expo-playground/app/components/providers/README.md

```
@@ -0,0 +1,119 @@
+if (res === false) {
+  throw new Error(t('sdk.initError'));
+}
+sdkInstance = HardwareWebSdk;
+
+// 设置事件监听器
+setupSDKEventListeners(sdkInstance);
+```
+
+## 组件层面的简化
+
+### TransportSwitcher
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)